src/content/post/drush-make-gitignore.md

```diff
@@ -0,0 +1,52 @@
++++
+date        = "2016-04-08T01:00:00-04:00"
```

Comment:
Ok, I've seen the way to do it. It's in config. I will leave the slug as is and open another PR for the config part